pages/services/edge-lb/1.0.0/release-notes/index.md

```diff
@@ -8,15 +8,53 @@ excerpt:
 enterprise: false
 ---
 
+# v1.0.0
+
+## Noteworthy changes:
+
+- Introducing V2 API with new more intuitive model for service / app backend selection model
+- New CLI command layout, ability to modify individual pools
+- Dockerized build and other CI improvements
+- Strict Mode Support
```

Comment:
State that strict mode support is only available in DC/OS Enterprise  v1.11 or higher.